src/test/kotlin/uk/gov/justice/digital/hmpps/whereabouts/services/PrisonApiServiceTest.kt

```diff
@@ -84,48 +81,6 @@ class PrisonApiServiceTest {
     )
   }
 
-  @Test
-  fun `update appointment comment - main flow`() {
-    val appointmentId = 100L
-    val comment = "New comment"
-
-    prisonApiMockServer.stubUpdateAppointmentComment(appointmentId)
-    prisonApiService.updateAppointmentComment(appointmentId, UpdateComment(comment), ALLOW)
-    prisonApiMockServer.verify(
-      putRequestedFor(urlEqualTo("/api/appointments/$appointmentId/comment"))
-        .withRequestBody(equalTo("""{"comment":"$comment"}"""))
-        .withHeader("no-event-propagation", equalTo("false")),
-    )
-  }
-
-  @Test
-  fun `update appointment comment - empty comment`() {
-    val appointmentId = 100L
-    val comment = ""
-
-    prisonApiMockServer.stubUpdateAppointmentComment(appointmentId)
-    prisonApiService.updateAppointmentComment(appointmentId, UpdateComment(comment), ALLOW)
-    prisonApiMockServer.verify(
-      putRequestedFor(urlEqualTo("/api/appointments/$appointmentId/comment"))
-        .withRequestBody(equalTo("""{"comment":""}"""))
-        .withHeader("no-event-propagation", equalTo("false")),
-    )
-  }
-
-  @Test
-  fun `update appointment comment - null comment`() {
-    val appointmentId = 100L
-    val comment = null
-
-    prisonApiMockServer.stubUpdateAppointmentComment(appointmentId)
-    prisonApiService.updateAppointmentComment(appointmentId, UpdateComment(comment), ALLOW)
-    prisonApiMockServer.verify(
-      putRequestedFor(urlEqualTo("/api/appointments/$appointmentId/comment"))
-        .withHeader("no-event-propagation", equalTo("false"))
-        .withRequestBody(equalTo("""{"comment":null}""")),
-    )
-  }
-
```

Comment:
This doesn't seem specific to VLB types - could apply to any type of appointment? Maybe its only used in the context of VLBs though ...

Reply:
Yes was only being used by VLB related code.